website/docs/configurations.md

```diff
@@ -3197,7 +3197,7 @@ Configurations that control compaction (merging of log files onto a new base fil
 ---
 
 > #### hoodie.keep.min.commits
-> Similar to hoodie.keep.max.commits, but controls the minimum number ofinstants to retain in the active timeline.<br></br>
+> Similar to hoodie.keep.max.commits, but controls the minimum number of instants to retain in the active timeline.<br></br>
```

Comment:
Please refrain from changing the `configurations.md` directly.  This is automatically generated and updated based on the Hudi config classes.

Reply:
I addressed it in #6542.

Reply:
@yihua Thanks for reviewing.